hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -122,18 +122,30 @@
     FRONTEND_SSL_PORT("frontend.ssl.port", 8443),
     FRONTEND_SSL_CLIENT_AUTH_MODE("frontend.ssl.client.auth.mode", "not_requested"),
     FRONTEND_SSL_PROTOCOL("frontend.ssl.protocol", "TLS"),
+
+    FRONTEND_SSL_KEYSTORE_DEFAULT_JVM("frontend.ssl.keystore.default.jvm", true),
+    FRONTEND_SSL_KEYSTORE_PROVIDED("frontend.ssl.keystore.provided", false),
     FRONTEND_SSL_KEYSTORE_LOCATION("frontend.ssl.keystore.location", "classpath:server.keystore"),
     FRONTEND_SSL_KEYSTORE_PASSWORD("frontend.ssl.keystore.password", "password"),
     FRONTEND_SSL_KEYSTORE_FORMAT("frontend.ssl.keystore.format", "JKS"),
+
+    FRONTEND_SSL_TRUSTSTORE_DEFAULT_JVM("frontend.ssl.truststore.default.jvm", true),
+    FRONTEND_SSL_TRUSTSTORE_PROVIDED("frontend.ssl.truststore.provided", false),
     FRONTEND_SSL_TRUSTSTORE_LOCATION("frontend.ssl.truststore.location", "classpath:server.truststore"),
     FRONTEND_SSL_TRUSTSTORE_PASSWORD("frontend.ssl.truststore.password", "password"),
     FRONTEND_SSL_TRUSTSTORE_FORMAT("frontend.ssl.truststore.format", "JKS"),
 
-    CONSUMER_SSL_ENABLED("consumer.ssl.enabled", false),
-    CONSUMER_SSL_PROTOCOL("consumer.ssl.protocol", "TLS"),
+    CONSUMER_SSL_ENABLED("consumer.ssl.enabled", true),
```

Comment:
Since property `consumer.http2.enabled` is enabled by default it should also be enabled.